ang/test/civicase/activity/factories/get-activity-feed-url.factory.spec.js

```diff
@@ -0,0 +1,151 @@
+/* eslint-env jasmine */
+
+(function (_, activityStatusTypes, extractQueryStringParams) {
+  describe('getActivityFeedUrl', function () {
+    var $location, $route, $sce, activityFeedUrlObject, getActivityFeedUrl;
+
+    beforeEach(module('civicase', function ($provide) {
+      $location = jasmine.createSpyObj('$location', ['path']);
+      $route = { current: { params: {} } };
+
+      $provide.value('$location', $location);
+      $provide.value('$route', $route);
+    }));
+
+    beforeEach(inject(function (_$sce_, _getActivityFeedUrl_) {
+      $sce = _$sce_;
+      getActivityFeedUrl = _getActivityFeedUrl_;
+    }));
+
+    describe('when getting the activity feed url from the dashboard page', function () {
+      beforeEach(function () {
+        $location.path.and.returnValue('/case');
+
+        activityFeedUrlObject = getActivityFeedUrlAsObject();
+      });
+
+      it('returns an url pointing to the dashboard activity feed', function () {
```

Comment:
Although the `U` at the first of URL is a vowel, the abbreviation is pronounced `you-are-ell`. So let's fix it to 
`returns a url pointing to the dashboard activity feed`